src/main/java/seedu/address/logic/commands/SortLexicographicalCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+
+/**
+ * Sort all persons in the address book by
+ * name in lexicographical order
+ */
+public class SortLexicographicalCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort-lexicographical";
```

Comment:
I feel that the command word is a bit complex for users. Having users to type sort-lexicographical seems a bit too much? Maybe you could shorten the command word?

Reply:
I agree, I like the shortened version, but personally for me I had to think a bit before understanding what lexicographical meant. Perhaps we could rename the command to something simpler so that foreign SoC Students can understand easier? Maybe other members can chime in.

Reply:
Thank you for your input. 
I am thinking of changing it to sort-name which simplifies the function and makes it easy for users to use as they need not understand what lexicographical means. I can further extend this to support other sorting fuctions like sort-module etc. What do you guys think? 🤔

Reply:
I like it!

Reply:
I think sort-name is more natural. Good idea!

Reply:
For sort-module you can implement it next week!

Reply:
I meant sort modules in a numerical order such as: